app/webpacker/controllers/producers_select_controller.js

```diff
@@ -0,0 +1,37 @@
+import TomSelect from "./tom_select_controller";
+
+const minimumOptionsForSearchField = 11;
+
+export default class extends TomSelect {
+  static values = { options: Object, placeholder: String };
+
+  connect(options = {}) {
+    const template = this.template();
+    let additional_options = {};
+    const lines_of_option = template.content.children.length; 
+
+    if (lines_of_option < minimumOptionsForSearchField) {
+      additional_options.plugins = [];  
+    }
+
+    super.connect(additional_options);
+    this.control.on('focus', this.fetchProducers.bind(this));
```

Comment:
It took me a while to figure out where this `control` was coming from. For anyone else, it's the `tom-select` controller, defined here : https://github.com/openfoodfoundation/openfoodnetwork/blob/master/app/webpacker/controllers/tom_select_controller.js

Reply:
I have added here a little comment too.